examples/dualstack/README.md

```diff
@@ -117,10 +117,8 @@ nginx-ipv6    LoadBalancer   fd00::6283       2603:1030:805:3::3   80:31140/TCP
 
 - Dual stack clusters are supported only with kubenet and azurecni.
   - Dual stack cluster with azurecni are only supported with `bridge` network mode.
-- Dual stack clusters are supported only with Linux.
-- Dual stack clusters with Windows is not supported at this time because it requires
-  - Kubernetes version 1.19+ and 
-  - [backport to 2004 to support dualstack containers](https://github.com/Azure/aks-engine/issues/3568).
+- Dual stack clusters are supported on Windows from version 2004 (kernel version 10.0.19041.610) and Kubernetes version 1.19
```

Comment:
Will dualstack also be supported on other windows versions? I think this should say "supported on Windows version ~from~ 2004"

Reply:
The Kubernetes docs says version 2004 or later, which was my understanding too - from 20H2 it would be a part of all Windows versions, but it was backported to 2004 from kernel version 10.0.19041.610.